q2_demux/tests/test_demux.py

```diff
@@ -680,6 +680,10 @@ def test_basic(self):
             png_fp = os.path.join(output_dir, 'demultiplex-summary.png')
             self.assertTrue(os.path.exists(png_fp))
             self.assertTrue(os.path.getsize(png_fp) > 0)
+            qual_forward_fp = os.path.join(output_dir,
```

Comment:
Looks good - can you also add something similar for the [paired-end test case](https://github.com/mahermassoud/q2-demux/blob/231d27342f5af4cfd6c6164556a3e551b5583389/q2_demux/tests/test_demux.py#L762) below?